tails_files/install.sh

```diff
@@ -53,7 +53,7 @@ cp securedrop_icon.png $INSTALL_DIR
 cp document.desktop $INSTALL_DIR
 cp source.desktop $INSTALL_DIR
 cp securedrop_init.py $SCRIPT_PY
-cp 99-tor-reload.sh $INSTALL_DIR
+cp 65-tor-reload.sh $INSTALL_DIR
```

Comment:
Minor request - can we name the hook so it clearly indicates it's part of the SecureDrop-specific customizations? e.g. "65-configure-tor-for-securedrop.sh`